plugins/terraform/provider.go

```diff
@@ -1,15 +1,15 @@
 package main
 
 import (
-	"time"
-
 	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
 	"github.com/quintilesims/layer0/cli/client"
+	"time"
 )
 
 var defaultTimeout = time.Minute * 15
 
-func Provider() *schema.Provider {
+func Provider() terraform.ResourceProvider {
```

Comment:
Is this change required? Wouldn't this cause the `TestProvider` function to fail at `if err := Provider().(*schema.Provider)`?